### ISSUE TITLE
is this still maintained?

### ISSUE THREAD

**Issue by graingert** [zaach/jsonlint#100]
@zaach ^

**Comment by graingert** [comment]
@zaach this project is now broken: https://github.com/npm/npm/issues/19875

**Comment by realityking** [comment]
See #103

**Comment by Berkmann18** [comment]
@graingert Seems like it.

**Comment by prantlf** [comment]
Try the NPM module [`paulvollmer/jsonlint`](https://www.npmjs.com/package/@paulvollmer/jsonlint) from [Paul Vollmer's fork](https://github.com/paulvollmer/jsonlint).

* It uses the new and maintained `nomnom` package to avoid audit problems.
* It fixes command-line parsing so that you can pass multiple files or use wildcards.

**Comment by Berkmann18** [comment]
@prantlf Just tried that one and I still get the warning.

**Comment by prantlf** [comment]
@Berkmann18, you're right, sorry. Audit reports no problems, but the package deprecation is reported as a non-fatal warning:

```
npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
```

**Comment by prantlf** [comment]
I needed some bugfixes and extensions to `jsonlint` and did them in [my fork](https://github.com/prantlf/jsonlint). I released the changes as a new NPM module [`@prantlf/jsonlint`](https://www.npmjs.com/package/@prantlf/jsonlint). Replacing the deprecated `nomnom` with `commander` in the package dependencies was one of them.
